src/transforms/log_to_metric.rs

```diff
@@ -401,7 +401,7 @@ impl FunctionTransform for LogToMetric {
                     })
                 }
                 Err(TransformError::TemplateRenderingError(error)) => {
-                    emit!(&TemplateRenderingFailed {
+                    emit!(&crate::internal_events::TemplateRenderingError {
```

Comment:
Could we just add `TemplateRenderingError` to the imports at the top of the file? It feels a bit weird to import some of the internal events there, but others with fully qualified names.

Reply:
we could but there is a conflict with an existing `TemplateRenderingError` which is the actual template rendering error.

Reply:
Ah, gotcha 👍